src/arrayMethodJoin.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
-/**
- * Implement method join
- */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let res = '';
```

Comment:
Ok, but it's a bad variable name, in this case `result` will be ok, but better to avoid words trimming